test/unit-test/main.tf

```diff
@@ -57,3 +57,56 @@ module "s3_with_notification" {
   notification_sns_arn = aws_sns_topic.topic.arn
   tags                 = local.tags
 }
+
+data "aws_caller_identity" "current" {}
+
+# KMS Source
+resource "aws_kms_key" "kms_primary_s3" {
+  description             = "kms_primary_s3"
+  policy                  = data.aws_iam_policy_document.kms_policy_s3.json
+  enable_key_rotation     = true
+  deletion_window_in_days = 30
+}
+resource "aws_kms_alias" "kms_primary_alias" {
+  name          = "alias/kms_primary_alias"
+  target_key_id = aws_kms_key.kms_primary_s3.id
+}
+data "aws_iam_policy_document" "kms_policy_s3" {
```

Comment:
Can this be moved out of `variables.tf` into either `main.tf`, or a dedicated `data.tf` file please?